examples/grasping/train_dqn_batch_grasping.py

```diff
@@ -206,7 +206,6 @@ def make_env(idx, test):
             maxSteps=max_episode_steps,
             isTest=test,
         )
-        assert env.observation_space is None
```

Comment:
I removed this line after this recent update to the environment: https://github.com/bulletphysics/bullet3/commit/3a6a279b9a757c4c1306fc55fa6e95df40c89574#diff-37edfb1b0750fd6a650da86f2d5c1ec3827623de1abb22ba2fc7cf897eeb5c44